database/db.js

```diff
@@ -9,7 +9,7 @@ const baseConfig = {
     // stop sequelize from pluralizing model names to get table names
     freezeTableName: true
   },
-  logging: ['development'].includes(env)
+  logging: false
```

Comment:
What's this about?

Reply:
You know... I'm not sure. I think I changed it so it wouldn't do unnecessary logging while I was working. I can change it back.